### PR TITLE
Guard isSupported()

### PR DESCRIPTION
**Pull request by purejava**
Fixes https://github.com/cryptomator/cryptomator/discussions/3476

### CODE REVIEW

**Comment by infeo** [comment]
@purejava I integrated a suggestion from the bot. (but apparently i was not careful enoguh 🙈 )

Anyway, thanks again for the in depth analysis of the problem and the quick fix!

**Comment by purejava** [comment]
> @purejava I integrated a suggestion from the bot. (but apparently i was not careful enoguh 🙈 )
> 

🙂 Such things happen.

> Anyway, thanks again for the in depth analysis of the problem and the quick fix!

You are welcome @infeo!

File: src/main/java/org/cryptomator/linux/keychain/SecretServiceKeychainAccess.java [hunk 1]
```diff
@@ -29,6 +29,9 @@ public String displayName() {
 	public boolean isSupported() {
 		try {
 			return SimpleCollection.isAvailable();
+		} catch (RuntimeException e) {
+			LOG.warn("Initializing secret service keychain access failed", e);
+			return false;
```
[the remark this answers is not in the document]
Comment (overheadhunter) [reply src/main/java/org/cryptomator/linux/keychain/SecretServiceKeychainAccess.java]: This is a terrible suggestion. @purejava @infeo please undo.

Obviously the blocks are not the same, there is a ternary operator needed within. We combine cases just to separate them again.

[the remark this answers is not in the document]
Comment (infeo) [reply src/main/java/org/cryptomator/linux/keychain/SecretServiceKeychainAccess.java]: I reverted my changes.

**Pull request merged**